resources/lang/fr/Order.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/*************************************************************************
+ Generated via "php artisan localization:missing" at 2018/04/25 15:38:13 
+*************************************************************************/
+
+return array (
+  //============================== New strings to translate ==============================//
+  // Defined in file C:\\wamp\\www\\attendize\\app\\Http\\Controllers\\EventOrdersController.php
+  'amount_refunded' => 'amount_refunded',
```

Comment:
?

Reply:
I was assuming that the string is not used in the english version, since "amount_refunded" does not seem to be UI-ready. What is the english phrase I should translate here?

Reply:
IDK maybe 'Total remboursé'

Reply:
I have no idea if it is used in the UI but it may be safer to have them translated now

Reply:
I'd rather stick to translating real english phrases for now.

@jeremyquinton do you know if those occurrences need to be translated in english, or are just ignored for now? The former and I'll translate them in french, the latter and I prefer not to.

Reply:
@bzg yes this is used.